LICENSE

```diff
@@ -4,41 +4,41 @@ License text copyright (c) 2020 MariaDB Corporation Ab, All Rights Reserved.
 Parameters
 
 Licensor:             HashiCorp, Inc.
-Licensed Work:        Nomdad Version 1.6.2 or later. The Licensed Work is (c) 2023
+Licensed Work:        Nomdad Version 1.7.0 or later. The Licensed Work is (c) 2023
```

Comment:
```suggestion
Licensed Work:        Nomad Version 1.7.0 or later. The Licensed Work is (c) 2023
```

Reply:
LOL, how did I miss that on the original? :laughing: